src/handlers/comment/action.ts

```diff
@@ -27,13 +28,21 @@ export const handleComment = async (): Promise<void> => {
     const userCommand = userCommands.find((i) => i.id == command);
 
     if (userCommand) {
-      const { handler, callback, successComment, failureComment } = userCommand;
+      const { id, handler, callback, successComment, failureComment } = userCommand;
       logger.info(`Running a comment handler: ${handler.name}`);
 
       const { payload: _payload } = getBotContext();
       const issue = (_payload as Payload).issue;
       if (!issue) continue;
 
+      const setting = config.command.find((e) => e.name === id.split("/")[1]);
+
+      if (!setting?.enabled) {
+        logger.info(`Skipping '${id}' because it is disabled on this repo`);
```

Comment:
I think it makes sense to name this variable "feature"

Reply:
Yea, it does.. :)